src/Std/Time/Zoned.lean

```diff
@@ -110,14 +111,14 @@ Converts a `PlainDate` to a `ZonedDateTime`.
 -/
 @[inline]
 def ofPlainDate (pd : PlainDate) (zr : TimeZone.ZoneRules) : ZonedDateTime :=
-  ZonedDateTime.ofPlainDateTime (pd.atTime PlainTime.midnight) zr
+  ZonedDateTime.ofPlainDateTime (pd.atTime (α := true) PlainTime.midnight) zr
```

Comment:
For example, these uses of `(α := true)` seem more readable as `(leap := true)`.